packages/admin/dashboard/src/routes/orders/order-allocate-items/components/order-create-fulfillment-form/utls.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  AdminProductVariant,
+  AdminProductVariantInventoryItemLink,
+  OrderLineItemDTO,
+} from "@medusajs/types"
+
+/**
+ * Check if the line item has inventory kit.
+ */
+export function checkInventoryKit(
+  item: OrderLineItemDTO & {
+    variant?: AdminProductVariant & {
+      inventory_items: AdminProductVariantInventoryItemLink[]
+    }
+  }
+) {
+  const variant = item.variant
+
+  if (!variant) {
+    return false
+  }
+
+  return (
+    (!!variant.inventory_items.length && variant.inventory_items.length > 1) ||
+    (variant.inventory_items.length === 1 &&
+      variant.inventory_items[0].required_quantity! > 1)
```

Comment:
question: Might be missing an obvious detail, but why is the second condition deemed an inventory kit?

Reply:
I was missing a detail; this makes sense. Thanks 😄